src/Promise/HttpFulfilledPromise.php

```diff
@@ -18,7 +18,7 @@ public function __construct(ResponseInterface $response)
         $this->response = $response;
     }
 
-    public function then(callable $onFulfilled = null, callable $onRejected = null)
+    public function then(?callable $onFulfilled = null, ?callable $onRejected = null)
```

Comment:
can we do this without the interface changing? if not, this bumps the minimum requirement of promise to the version that will contains https://github.com/php-http/promise/pull/34